pkg/connectors/wizio/connector.go

```diff
@@ -0,0 +1,235 @@
+package wizio
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"github.com/synyx/tuwat/pkg/connectors"
+	"github.com/synyx/tuwat/pkg/connectors/common"
+	html "html/template"
+	"io"
+	"log/slog"
+	"net/http"
+)
+
+type Connector struct {
+	config Config
+	client *http.Client
+}
+
+type Config struct {
+	Tag            string
+	StatusFilter   []string
+	SeverityFilter []string
+	common.HTTPConfig
+	NumberOfIssues int
+}
+
+func NewConnector(cfg *Config) *Connector {
+	return &Connector{*cfg, cfg.HTTPConfig.Client()}
```

Comment:
Instead of needing the user to configure the `audience = "wiz-io"` (thus necessitating a comment for the end-user), how about we add the config here.

```suggestion
  cfg.HTTPConfig.OAuth2Creds.EndpointParams["audience"] = "wiz-io"
	return &Connector{*cfg, cfg.HTTPConfig.Client()}
```

Reply:
Thanks, I was somehow blind to the possibility.
`cfg.OAuth2Creds.EndpointParams["audience"] = []string{"wiz-api"}`